Assets/Scripts/Blocks/LockBlock.cs

```diff
@@ -52,6 +59,9 @@ protected override void OnTouch(GameObject player)
         // If the the lockBlock is unlocked
         if (!((LockBlock_Data)BlockData).GetLock())
         {
+            // Play sound effect
+            audioSource.PlayOneShot(audioClip, Random.Range(0.5f, 1.5f));
```

Comment:
Why random volumes?

Reply:
Otherwise it sounds always the same which is boring if you touch / destory a lot of blocks, this way the sound of each blocks is slightly different, which for users sound more natural as sounds in real life don't always sound the same.